### ISSUE TITLE
INSERT DATA query on solid pod throws UnsupportedMediaTypeHttpError

### ISSUE THREAD

**Issue by pbonte** [comunica/comunica-feature-solid#31]
#### Issue type:

- :bug: Bug 

____
#### Description:

When inserting data using the SPARQL INSERT DATA clause on a solid pod, an UnsupportedMediaTypeHttpError is thrown. Inspecting the payload of the message sent from comunica to the solid server shows that a stream object `[object ReadableStream]` is transmitted (instead of the content of the stream).
The following code fragment was used to insert the data:
```
await this.comunica.queryVoid("INSERT DATA { <ex:s> <ex:p> <ex:o> }", {
				sources: [this.baseUrl+"todo.ttl"],
				"@comunica/actor-http-inrupt-solid-client-authn:session": getDefaultSession(),
		});
```
Fixing the destination types did not solve the problem.
____
#### Environment:

| software         | version
| ---------------- | -------
| Comunica Engine  | 2.0.6 
| node             | v16.13.2
| npm              | 8.1.2
| yarn             | 1.22.10
| Operating System | darwin (Darwin 20.6.0)
#### Crash log:

https://pastebin.com/EFNFhAR7

**Comment by rubensworks** [comment]
I just tested this within my pod on solidcommunity.net via Node.js, and everything seems to be working just fine.

Could you provide additional details on what you're doing differently?

**Comment by rubensworks** [comment]
Ok, so the root cause of this issue is that apparently browsers don't yet support passing `ReadableStream` as body to `fetch`, and they just stringify it instead:
* https://bugs.chromium.org/p/chromium/issues/detail?id=688906
* https://bugzilla.mozilla.org/show_bug.cgi?id=1387483

So I'll look into a temporary workaround.

**Comment by rubensworks** [comment]
Fixed in Comunica 2.2.1. (Might require a lockfile+node_modules reset)
